### PR TITLE
Veda branch

### PR DESCRIPTION
**Pull request by Pavubrar**

### CODE REVIEW

**Comment by Pavubrar** [comment]
please revise changes

**Review by Pavubrar — COMMENTED** [review]
good to merge product.jsx and fetch.jsx

File: pluralistic/src/components/Navbar.css
Comment (Pavubrar) [inline pluralistic/src/components/Navbar.css:1]: no more needed this file

File: pluralistic/src/components/Navbar.jsx
Comment (Pavubrar) [inline pluralistic/src/components/Navbar.jsx:1]: already updated this file

**Review by vedavyasananthasetty — APPROVED** [review]
There are still conflicts

**Review by Pavubrar — COMMENTED** [review]
good to merge product.jsx and fetch.jsx

**Pull request closed without merging**